benchmarks/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
 }
 
 task qps_client(type: CreateStartScripts) {
-    mainClassName = "io.grpc.benchmarks.qps.QpsClient"
+    mainClassName = "io.grpc.benchmarks.qps.QpsAsyncClient"
```

Comment:
As we discussed, I do think we will want a single binary that supports both sync and async.